ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rhcs.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    release:
+      channel: candidate
+      version: "4.14"
```

Comment:
DO we define the version in PROW or cluster-profile? Currently it is set in profile